WalletWasabi.Backend/Global.cs

```diff
@@ -105,10 +105,6 @@ public async Task InitializeAsync(CoordinatorRoundConfig roundConfig, Cancellati
 				{
 					throw new ArgumentException($"Blacklist API token was not provided in {nameof(WabiSabiConfig)}.");
 				}
-				if (CoordinatorParameters.RuntimeCoordinatorConfig.RiskFlags is null || !CoordinatorParameters.RuntimeCoordinatorConfig.RiskFlags.Any())
```

Comment:
No no. You should never ever allow garbage get in, otherwise you would need to check by null before every single fucking  reference. Are you doing that? No, nobody do that (thanks God). If `RiskFlags` is `null` then there is something horrible wrong that have to be fixed.